adr/_readme.md

```diff
@@ -0,0 +1,4 @@
+Conventions for ADRs:
```

Comment:
Shouldn't we keep the README.md convention for this file name? It will always be at the end of the file.